README.md

```diff
@@ -276,10 +276,11 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 
 *Packages that help with building Distributed Systems.*
 
+* [dht](https://github.com/anacrolix/torrent/dht) - BitTorrent Kademlia DHT implementation.
```

Comment:
This link is 404.

You either meant to link to https://github.com/anacrolix/torrent/tree/master/dht, or https://godoc.org/github.com/anacrolix/torrent/dht.